system-test/fixtures/kitchen/src/index.ts

```diff
@@ -4,7 +4,7 @@ import {GoogleAuthOptions, Operation,Service, ServiceConfig, ServiceOptions,
   ServiceObjectConfig, StreamRequestOptions, Abortable, AbortableDuplex,
   ApiError, util} from '@google-cloud/common';
 
-util.makeRequest({url: 'test'}, {}, (err, body, res) => {
+util.makeRequest({uri: 'test'}, {}, (err, body, res) => {
```

Comment:
@JustinBeckwith is this cool? This was the sole fix for all of that monospace sadness.

Reply:
s'all good